### ISSUE TITLE
extra features

### ISSUE THREAD

**Issue by Hessel91** [pcvonz/oc_displaycase_plugin#1]
Hi, 

Thank you for your work. However I'm missing a next and prev on the single project page wich switches to the next project item. 
And is it possible to add a category option on the landingpage component based on the tag option  for example isotope.js ?

regards, 

Hessel

**Comment by pcvonz** [comment]
Next and previous buttons would be an easy addition. I'll look into it. 
Can you explain the second one a little more, having trouble understanding what you mean.

**Comment by Hessel91** [comment]
great thank you!

Oké I'll try to explain.
- It would be great to have some sort of filter included. For example sort the projects in the landingspage component by categorie or year. One of the filter methods u can use is isotope > https://isotope.metafizzy.co/.

**Comment by pcvonz** [comment]
Ohh, ok. That wouldn't be too hard either. You can use the reorder button on the item page in the backend for now though.

**Comment by Hessel91** [comment]
Yes but I mean on the Front-end, sorry for not being clear in my previous answer.

**Comment by pcvonz** [comment]
![screenshot_20171104_170609](https://user-images.githubusercontent.com/6378569/32410626-788e3866-c182-11e7-9b33-a1508e8e8e56.png)
Is this kind of what you're talking about? Or do you mean something a person visiting your site could use to sort projects?

**Comment by pcvonz** [comment]
3224edfa07af144fa5e0f5bb27b7dff764ac93a9
This commit addresses sorting the landing page from the backend and navigating to the previous and next portfolio item in the itemPage component. 
I'll update the plugin in the marketplace if this works for your use case. I'd like to test it out more too.

**Comment by Hessel91** [comment]
Hi Paul, sorry my late respons. 

yes I mean a person visiting the website could sort the projects in the landingpage component.

I didn't manage to get the commited version working. 
I installed the landingpage component on my homepage, and called on of the case items: new item.
but when you click on it there is something wrong with the url, look:
 https://mywebsitename/projects/new-item/title/website > and I get 404.

and if i insert the correct url of the project: https://mywebsitename/projects/new-item
then this happends
![screenshot-hesselkers nl 2017-11-11 12-26-09-081](https://user-images.githubusercontent.com/7326568/32688997-8b5caf8c-c6db-11e7-9ca5-e1895754de63.png)

perhaps when you have time you can have a look at it.
